### PR TITLE
update OCP sidecar images for Z support

### PR DESCRIPTION
**Pull request by oriyarde**
change sidecar images in OLM to support Z-systems

### CODE REVIEW

File: deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -72,26 +74,26 @@ metadata:
...
-                "tag": "v1.2.0",
+                "repository": "registry.redhat.io/openshift4/ose-csi-driver-registrar",
+                "tag": "v4.3",
                 "imagePullPolicy": "IfNotPresent"
               },
               {
                 "name": "csi-provisioner",
-                "repository": "quay.io/k8scsi/csi-provisioner",
-                "tag": "v1.3.0",
+                "repository": "registry.redhat.io/openshift4/ose-csi-external-provisioner-rhel7",
```
Comment (27149chen) [inline deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml]: did you test all the new sidecars?

File: deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -35,7 +35,8 @@ metadata:
                             "key": "kubernetes.io/arch",
                             "operator": "In",
                             "values": [
-                              "amd64"
+                              "amd64",
+                              "s390x"
```
Comment (27149chen) [inline deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml]: do we need to support Z for openshift in this release? AFAIK, ocp on Z is different versioning with x86.

File: deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml (hunk 1 above)
Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml]: This branch is used by the tests, I will merge it once they will pass

File: deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml (hunk 2 above)
Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml]: We plan to, yes. whether if they are different or not, we will test both of them.

File: deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -72,26 +74,26 @@ metadata:
             "sidecars": [
               {
                 "name": "csi-node-driver-registrar",
-                "repository": "quay.io/k8scsi/csi-node-driver-registrar",
-                "tag": "v1.2.0",
+                "repository": "registry.redhat.io/openshift4/ose-csi-driver-registrar",
```
Comment (27149chen) [inline deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml]: does it mean that we will store our controller and node images on redhat registry too?

Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml]: it sounds likely, but not sure if so in 1.1.0

Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml]: @27149chen can we resolve this?

Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml]: is it also one click with multi-arch images?

Comment (27149chen) [reply deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml]: you need to cert the images one by one (one image per one project), but I'm not sure if the redhat cert env supports z or power.

Comment (27149chen) [reply deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml]: And we you finish the cert, you just need one click.

Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml]: I'm not sure I follow.
Assuming they support Z, Where do I push the multi-arch manifest? do you have an example from ubiquity?

Comment (27149chen) [reply deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml]: what is the multi-arch manifest? I think we just need to push the images with different arch to the registry which support multi-arch, isn't it?

Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml]: https://docs.docker.com/engine/reference/commandline/manifest/#extended-description
I think there is more to it

Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml]: It seems `registry.connect.redhat.com/ibm` doesn't support this "v2 schema" at the moment, so I guess we'll have to change our operator repository to dockerhub, until they do

Comment (27149chen) [reply deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml]: but as far as i know, we have to push the operator image to redhat registry

Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml]: Then I will ask Scale how they did it

**Pull request merged**